Styles/main.css

```diff
@@ -2,6 +2,7 @@
 @import url(/Styles/products.css);
 @import url(/Styles/landing.css);
 @import url(/Styles/colors.css);
+@import url(/Styles/wishlist.css);
 * {
   box-sizing: border-box;
 }
```

Comment:
Aren't these already imported from your library?

Reply:
nope, i didn't use them in the component library